src/csm_testing/tests/sat_functional/hwinv/test_hwinv.py

```diff
@@ -0,0 +1,630 @@
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+"""
+Simple tests for the functionality of the `sat hwinv` command.
+"""
+import shlex
+import subprocess
+import unittest
+from typing import List
+
+HWINV_NODE_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'SKU', 'Serial Number',
+    'Cabinet Type', 'Memory Type', 'Memory Device Type', 'Memory Manufacturer',
+    'Memory Size (GiB)', 'Memory Module Count', 'Processor Count', 'Processor Manufacturer',
+    'Processor Model', 'Accelerator Count', 'Accelerator Riser Count', 'HSN NIC Count',
+    'Drive Count', 'Total Drive Capacity (GiB)', 'BIOS Version', 'Memory Model'
+]
+HWINV_CHASSIS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
+]
+HWINV_HSN_BOARDS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
+]
+HWINV_COMPUTE_MODULES_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
+]
+HWINV_ROUTER_MODULES_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
+]
+HWINV_NODE_ENCLOSURES_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'SKU', 'Serial Number'
+]
+HWINV_NODE__ENCLOSURES_POWER_SUPPLIES_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
+]
+HWINV_PROCS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Total Cores', 'Total Threads', 'Max Speed (MHz)'
+]
+HWINV_NODE_ACCELS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Location Name'
+]
+HWINV_NODE_ACCEL_RISERS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Board Serial Number', 'Producer', 'Engineering Change Level'
+]
+HWINV_NODE_HSN_NICS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
+]
+HWINV_MEMS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Memory Type', 'Device Type', 'Capacity (MiB)', 'Operating Speed (MHz)'
+]
+HWINV_DRIVES_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Media Type', 'Capacity (GiB)', 'Percent Life Left'
+]
+HWINV_CMM_RECTIFIERS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Power Supply Type', 'Firmware Version', 'Power Input Watts', 'Power Output Watts'
```

Comment:
I think this is the correct order:

```suggestion
    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'SKU', 'Serial Number',
    'Power Input Watts', 'Power Output Watts', 'Power Supply Type', 'Firmware Version'
```